graf2d/gpad/inc/TPad.h

```diff
@@ -307,7 +307,7 @@ class TPad : public TVirtualPad, public TAttBBox2D {
    void              Range(Double_t x1, Double_t y1, Double_t x2, Double_t y2) override; // *MENU* *ARGS={x1=>fX1,y1=>fY1,x2=>fX2,y2=>fY2}
    virtual void      RangeChanged() { Emit("RangeChanged()"); } // *SIGNAL*
    void              RangeAxis(Double_t xmin, Double_t ymin, Double_t xmax, Double_t ymax) override;
-   virtual void      RangeAxisChanged() { Emit("RangeAxisChanged()"); } // *SIGNAL*
+   virtual void      RangeAxisChanged() override { Emit("RangeAxisChanged()"); } // *SIGNAL*
```

Comment:
As the implementation of RangeAxisChanged is nos in TVirtualPad, you do not need to have it in TPad. Like UnZoomed